src/Shell/Task/PluginTask.php

```diff
@@ -235,7 +237,7 @@ protected function _modifyAutoloader($plugin, $path)
             $cwd = getcwd();
 
             // Windows makes running multiple commands at once hard.
-            chdir(dirname($path));
+            chdir(ROOT);
```

Comment:
That's totally fine, I will update the code to use the function tomorrow. Thanks for your advice @AD7six